hvantk/ptm/plot.py

```
@@ -98,65 +95,36 @@ def plot_landscape_summary(
-        f"{result.enrichment_ci_high:.2f}" if result.enrichment_ci_high < 1e6 else "∞"
+        f"{result.enrichment_ci_high:.2f}"
+        if result.enrichment_ci_high < 1e6 else "∞"
     )
     ax1.text(
-        0.98,
-        0.95,
+        0.98, 0.95,
         "PTM site + proximal vs non-PTM\n"
         f"OR={result.enrichment_odds_ratio:.2f} "
-        f"({result.enrichment_ci_low:.2f}-{ci_hi_str})\n"
+        f"({result.enrichment_ci_low:.2f}–{ci_hi_str})\n"
```

Ruff (0.15.7)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)